tests/providers/aws/services/cloudwatch/cloudwatch_log_group_no_secrets_in_logs/cloudwatch_log_group_no_secrets_in_logs_test.py

```diff
@@ -11,6 +11,13 @@
     set_mocked_aws_provider,
 )
 
+timestamp = int(unix_time_millis())
+dttimestamp = (
```

Comment:
What is the purpose of this one?

Reply:
timestamp is required to be milliseconds when submitting the put_log_events: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/logs/client/put_log_events.html#put-log-events

dttimestamp is set to build the timestamp to perform the comparison in the results.

Reply:
Thanks for clarifying this!